src/test/java/com/qa/koel/Homework21Test.java

```diff
@@ -28,15 +34,15 @@ public void renamePlaylistTest() throws InterruptedException {
 
         wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li[contains(text(), 'Edit')]")));
 
-        WebElement editButtonClick = getDriver().findElement(By.xpath("//li[contains(text(), 'Edit')]"));
-        editButtonClick.click();
+        playlistPage = new PlaylistPage(getDriver());
+        playlistPage.getEditButtonClick().click();
 
         wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[name='name']")));
-        WebElement playListNameInputField = getDriver().findElement(By.cssSelector("[name='name']"));
+        playlistPage.getPlayListNameInputField();
 
-        actions.moveToElement(playListNameInputField)
+        actions.moveToElement(playlistPage.getPlayListNameInputField())
                 .doubleClick().perform();
-        actions.moveToElement(playListNameInputField)
+        actions.moveToElement(playlistPage.getPlayListNameInputField())
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   For example

![Clean_Test](https://github.com/Dmussu/A50-QA-Automation-Framework/assets/104761482/5223e4b5-beaa-4a08-91ed-8424c4f82f9d)